package.json

```diff
@@ -7,17 +7,20 @@
     "url": "https://github.com/openfisca/openfisca-doc.git"
   },
   "scripts": {
+    "prebuild": "npm run clean",
     "build": "gitbook build",
-    "clean": "rm -rf _book",
+    "clean": "del _book",
```

Comment:
@cbenz what is this `del` ? Is someone working on windows somewhere ? I can't run the tests !